src/styles/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+  margin-top: 16px;
+
+  &__star {
+    background: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-position: center;
+
+    & :last-child() {
+      margin-right: 0;
+    }
+  }
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+      background: url(../images/star-active.svg);
+      background-repeat: no-repeat;
+      background-position: center;
```

Comment:
no need to repeat all styles in the loop

Reply:
Not resolved